src/components/JsonSchemaValidator/dbtFractribution.js

```diff
@@ -0,0 +1,119 @@
+export const dbtSnowplowFractributionConfigSchema = {
+  type: 'object',
+  properties: {
+    snowplow__page_views_source: {
+      type: 'string',
+      title: 'Page Views Source',
+      description: 'Source of the snowplow_web_page_views table',
+    },
+    snowplow__web_user_mapping_table: {
+      type: 'string',
+      title: 'User Mapping Table',
+      description: 'Schema and table of the User Mapping table',
+    },
+    snowplow__conversions_source: {
+      type: 'string',
+      title: 'Conversions Source',
+      description: 'Source of your conversions table',
+    },
+    snowplow__conversion_window_start_date: {
+      type: 'string',
+      format: 'date',
+      title: 'Conversions Window Start Date',
+      description: 'Start date (UTC) of your conversion window',
+    },
+    snowplow__conversion_window_end_date: {
+      type: 'string',
+      format: 'date',
+      title: 'Conversions Window End Date',
+      description: 'End date (UTC) of your conversion window',
+    },
+    snowplow__path_lookback_days: {
+      type: 'integer',
+      title: 'Path Lookback (days)',
+      description:
+        'Days to lookback from the conversion to include in the path',
+      minimum: 1,
+    },
+    snowplow__path_lookback_steps: {
+      type: 'integer',
+      title: 'Path Lookback Steps',
+      description:
+        'Maximum number of steps before the conversion (0 is unlimited)',
+      minimum: 0,
+    },
+    snowplow__path_transforms: {
+      type: 'object',
+      properties: {
+        exposure_path: {
+          type: 'string',
+          title: 'Exposure path',
+          description: 'Consequitive same channels are reduced to one',
+          enum: ['null'],
+        },
+        first_path: {
+          type: 'string',
+          title: 'First Path',
+          description: 'First occurance of path is kept',
+          enum: ['null'],
+        },
+        frequency_path: {
+          type: 'string',
+          title: 'Frequency Path',
+          description: 'Keeps count of channel frequency',
+          enum: ['null'],
+        },
+        remove_if_last_and_not_all: {
+          type: 'string',
+          title: 'Remove If Last And Not All',
+          description:
+            'Remove the channel if it is the last, but not all, channel',
+        },
+        remove_if_not_all: {
+          type: 'string',
+          title: 'Remove If Not All',
+          description:
+            "Remove all occurances of the channel, if it isn't the only channel",
+        },
+        unique_path: {
+          type: 'string',
+          title: 'Unique Path',
+          description: 'No change',
+          enum: ['null'],
+        },
+      },
+    },
```

Comment:
enum of only null on most of these doesn't seem right?

Reply:
You'd think so, but the thing is most of them don't actually have any parameters - however if I don't give them the option to have _some_ value I can't get them to render in the output. An empty string is also wrong, so this was the best way I could find to get it to work...

Reply:
I did however not correctly understand how types and values work in jsonschema, which is now fixed